didadata/migrations/0001_initial.py

```diff
@@ -27,7 +27,7 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(primary_key=True, verbose_name='ID', auto_created=True, serialize=False)),
                 ('timestamp', models.DateTimeField(auto_now_add=True, db_index=True, verbose_name='Timestamp')),
                 ('value', models.FloatField(verbose_name='Value')),
-                ('metric', models.ForeignKey(verbose_name='Metric', to='didadata.Metric', on_delete=models.deletion.CASCADE)),
+                ('metric', models.ForeignKey(verbose_name='Metric', to='didadata.Metric', on_delete=models.PROTECT)),
```

Comment:
Why not cascade?

Reply:
It could be possible that there are a huge amount of data and if I delete the metric all records are lost. That was the reason why I changed it. Should I revert it?

Reply:
I set it back to cascade. You´r right. If I decide to delete a metric, I don´t want to have a huge amount of data in my db.